DB/classification/models.py

```diff
@@ -57,6 +52,32 @@ def __init__(self, model_type="default", created_at=datetime.datetime.utcnow(),
         self.model = model
         self.id = _id
 
+    @classmethod
+    def reload_db_info(cls):
+        """
+        :summary: Reloads which database/user pair to use (for testing). Defaults to seads/seadapi when no file is present (old behavior)
+        """
+        db_info = None
+        try:
+            db_info = open("db_info", 'r')
```

Comment:
Stub out the actual database calls and test that the correct queries and data are executed.

See these tests:
https://github.com/seadsystem/Backend/blob/master/DB/api/test_query_db.py